### DISCUSSION TITLE
Compatibility with Expo Web

### DISCUSSION THREAD

**Discussion by cliftonlabrum** [bvaughn/react-resizable-panels#270] [Q&A] [answered]
I'm new to React and am using Expo to build my first app. I am trying to use this package in the web version of my Expo app, but it doesn't seem to work.

```jsx
//app/index.tsx
import { PanelGroup, Panel, PanelResizeHandle } from 'react-resizable-panels';

export default function Page() {  
  return (
    <PanelGroup direction='horizontal'>
      <Panel defaultSize={30} minSize={20}>
        left
      </Panel>
      <PanelResizeHandle />
      <Panel minSize={30}>
        middle
      </Panel>
      <PanelResizeHandle />
      <Panel defaultSize={30} minSize={20}>
        right
      </Panel>
    </PanelGroup>
  );
}
```
My assumption is that it won't work because this package is specific to React JS/Web and not React Native. Is that correct? Or am I missing something?

**Comment by bvaughn** [comment, marked as answer]
The project does specific a `react-dom` peer dependency because it renders DIV elements. I don't know if another renderer _could_ be mad to work if you specified `tagName` overrides. Supporting Expo Web is not something I have the bandwidth to do though probably.
